post_office/models.py

```diff
@@ -138,6 +143,11 @@ def dispatch(self, log_level=None, disconnect_after_delivery=True):
         self.status = status
         self.save()
 
+        if status == STATUS.failed:
+            logger.error("email send fail: %s", self)
```

Comment:
The log level for this should be "info" instead of error because this is not an error in this library. Also, please change the message to "Delivery failed: "